tests/terraform/checks/test_multiple_skips.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from pathlib import Path
+
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.runner import Runner
+
+
+class TestMultipleSkips(unittest.TestCase):
+
+    def test(self) -> None:
+        # given
+        test_files_dir = Path(__file__).parent / "a example skip"
+
+        # when
+        report = Runner().run(root_folder=str(test_files_dir), runner_filter=RunnerFilter(checks=[]))
+
+        # then
+        summary = report.get_summary()
```

Comment:
Please assert the skipped part in the reports by resource - 
We want to be sure `default` and `skip_invalid` are with one skip, and `skip_more_than_one` is with 2 skips.